### PR TITLE
 Numerical methods for calculate integral in C++

### PR DESCRIPTION
**Pull request by OkarinRintaro**
Changes:  

Created new section: Numerical analysis  
Created integral_rectangle.cpp  
Created integral_trapezoid.cpp

### CODE REVIEW

File: code/numerical_analysis/integral/integral_rectangle.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include <functional>
+
+//Part of Cosmos by OpenGenus Foundation
+double f(double x)
```
Comment (arnavb) [inline code/numerical_analysis/integral/integral_rectangle.cpp]: What does this function do? Why is this even necessary in your code?

Comment (OkarinRintaro) [reply code/numerical_analysis/integral/integral_rectangle.cpp]: f is just example for examination to work of integral. Better delete?

Comment (OkarinRintaro) [reply code/numerical_analysis/integral/integral_rectangle.cpp]: f is symbol of mathematics function.

Comment (arnavb) [reply code/numerical_analysis/integral/integral_rectangle.cpp]: Oh, I see, but it seems somewhat redundant, since this is code, anyways.

**Comment by OkarinRintaro** [comment]
done it

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**